docs/man/man1/openshift-infra-f5-router.1

```diff
@@ -24,6 +24,14 @@ You may restrict the set of routes exposed to a single project (with \-\-namespa
 
 .SH OPTIONS
 .PP
+\fB\-\-allow\-wildcard\-routes\fP=false
+    Allow wildcard host names for routes
+
+.PP
+\fB\-\-allowed\-domains\fP=[]
```

Comment:
Do we need some kind of depth restriction for wildcards to prevent someone from trying to claim *.com and other TLDs?  Not as a fixed number, but as an env with a default?